container/test-app/index.html

```diff
@@ -1,670 +1,76 @@
-            // send back to mf alert.answer with data
-            window.postMessage('alert.answer', data);
-            event.luigiCallback(val);
-          }
-        );
-        luigiContainer.addEventListener(MFEventID.CUSTOM_MESSAGE, event => {
-          console.log('Custom Event', event);
-          if (event.detail.id !== 'timer') {
-            alert(event.detail.id);
-          }
-        });
-
-        luigiContainer.addEventListener(
-          MFEventID.RUNTIME_ERROR_HANDLING_REQUEST,
-          event => {
-            console.log(event.detail);
-          }
-        );
-        luigiContainer.addEventListener(MFEventID.ADD_NODE_PARAMS_REQUEST, event => {
-          const params = event.detail.params;
-          console.log('params', params);
-        });
-        luigiContainer.addEventListener(MFEventID.SET_ANCHOR_LINK_REQUEST, event => {
-          console.log('anhor', event.detail);
-        });
-
-        // uxManager(). closeUserSettings openUserSettings collapseLeftSideNav
-        // setDocumentTitle removeBackdrop hideAppLoadingIndicator
-        luigiContainer.addEventListener(MFEventID.CLOSE_USER_SETTINGS_REQUEST, event => {
-          console.log('Close User Settings Request received', event.detail);
-        });
-        luigiContainer.addEventListener(MFEventID.OPEN_USER_SETTINGS_REQUEST, event => {
-          console.log('Open User Settings Request received', event.detail);
-        });
-        luigiContainer.addEventListener(MFEventID.COLLAPSE_LEFT_NAV_REQUEST, event => {
-          console.log('Collapse Left Side Nav Request received', event.detail);
-        });
-        luigiContainer.addEventListener(MFEventID.SET_DOCUMENT_TITLE_REQUEST, event => {
-          console.log('Set Document Title Request received', event.detail);
-        });
-        luigiContainer.addEventListener(MFEventID.REMOVE_BACKDROP_REQUEST, event => {
-          console.log('Remove Backdrop Request received', event.detail);
-        });
-        luigiContainer.addEventListener(
-          MFEventID.HIDE_LOADING_INDICATOR_REQUEST,
-          event => {
-            console.log('Hide Loading Indicator Request received', event.detail);
-          }
-        );
-
-        // linkManager listeners:
-        // path exists
-        luigiContainer.addEventListener(MFEventID.PATH_EXISTS_REQUEST, event => {
-          console.log('Remove Backdrop Request received', event.detail, event);
-          // send back result with defined 'callback'
-          // event: MFEventID.PathExistsEvent can be used as an event type to get the callback function
-          event.callback(true);
-        });
-
-        // setViewGroup Data listener
-        luigiContainer.addEventListener(MFEventID.SET_VIEW_GROUP_DATA_REQUEST, event => {
-          console.log('Set View Group Data Request received', event.detail, event);
-        });
-      });
-      let testDeferInitFalse = document.getElementById('test-defer-init-false');
```

Comment:
We have no example for this flag in the new test suite. Should we add one?

Reply:
Yes. this one and maybe others too. I will create a Follow-Up dedicated to only adding tests for the new suite. This PR is already too big.